ui/packages/shared/profile/src/ProfileView/index.tsx

```diff
@@ -52,7 +53,8 @@ type NavigateFunction = (path: string, queryParams: any, options?: {replace?: bo
 
 export interface FlamegraphData {
   loading: boolean;
-  data?: Flamegraph;
```

Comment:
We might need to cater for this removed prop for the grafana plugin as that's the reason for the typing error [here](https://github.com/parca-dev/parca/actions/runs/5457111282/jobs/9930779290?pr=3326).

Reply:
aah, I see. Let's not rename it then, but only add the table one for now. I think there is a bigger refactoring we should do anyway.